ModelBundle/Tests/Validator/Constraints/IdAuthorizedCharacterValidatorTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Tests\Validator\Constraints;
+
+use Phake;
+use OpenOrchestra\ModelBundle\Validator\Constraints\IdAuthorizedCharacter;
+use OpenOrchestra\ModelBundle\Validator\Constraints\IdAuthorizedCharacterValidator;
+
+/**
+ * Class IdAuthorizedCharacterValidatorTest
+ */
+class IdAuthorizedCharacterValidatorTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var IdAuthorizedCharacterValidator
+     */
+    protected $validator;
+
+    protected $context;
+    protected $constraint;
+    protected $constraintViolationBuilder;
+
+    /**
+     * Set up the test
+     */
+    public function setUp()
+    {
+        $this->constraint = new IdAuthorizedCharacter();
```

Comment:
you could also phake it. As Phake is extending your class, you will still have access to the public properties